documentation/source/user_section/installation/windows.rst

```diff
@@ -4,60 +4,38 @@
 Installation for Windows
 ************************
 
```

Comment:
## Discussion: SCT Course

This PR needs to be merged and included in a stable release for it to be available for our users prior to the SCT Course. (This means v5.5). 

However, that would mean creating a new release with Python 3.7, which runs right into the M1 Mac issue (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3566). Now, we have a almost-working fix for that too (in #3569), but the `grep` issue in that PR has yet to be addressed.

So... as far as making sure SCT is usable for both Windows and Mac M1 users... we could either:

* Backport these fixes to SCT v5.4, quietly, amend that release, and stick with Python 3.6/Tensorflow 1.5.
* Try to get #3569 ready and merged, then create a new release ASAP.

Reply:
> Backport these fixes to SCT v5.4, quietly amend that release, and stick with Python 3.6/Tensorflow 1.5.

I would feel more at ease with this solution because *think* it will create the least collateral damage few days before the SCT course. The risk with going with a new version (option 2) is that it has not been thoroughly tested, and there is much more actions to do (more stressful, more prone to errors).